polaris-tokens/src/token-groups/color.ts

```diff
@@ -767,61 +767,61 @@ export const color: {
   },
   // Experimental tokens
   'color-bg-backdrop-experimental': {
-    value: colorsExperimental.gray[16]('0.75'),
+    value: colorsExperimental.blackAlpha[14],
     description: '',
   },
   'color-bg-primary-disabled-experimental': {
-    value: colorsExperimental.gray[9](),
+    value: colorsExperimental.gray[9],
     description: '',
   },
   'color-bg-secondary-experimental': {
-    value: colorsExperimental.gray[5](),
+    value: colorsExperimental.gray[5],
     description: '',
   },
   'color-bg-input-hover-experimental': {
-    value: colorsExperimental.gray[3](),
+    value: colorsExperimental.gray[3],
     description: '',
   },
   'color-border-input-active-experimental': {
-    value: colorsExperimental.gray[16](),
+    value: colorsExperimental.gray[16],
   },
   'color-border-critical-strong-experimental': {
     value: colorsExperimental.red[14],
   },
   'color-bg-input-active-experimental': {
-    value: colorsExperimental.gray[4](),
+    value: colorsExperimental.gray[4],
     description: '',
   },
   'color-bg-transparent-experimental': {
-    value: colorsExperimental.gray[16]('0'),
+    value: colorsExperimental.blackAlpha[1],
     description: '',
   },
   'color-bg-transparent-subdued-experimental': {
-    value: colorsExperimental.gray[16]('0.07'),
+    value: colorsExperimental.blackAlpha[7],
     description: '',
   },
   'color-bg-transparent-hover-experimental': {
-    value: colorsExperimental.gray[16]('0.05'),
+    value: colorsExperimental.blackAlpha[5],
     description: '',
   },
   'color-bg-transparent-active-experimental': {
-    value: colorsExperimental.gray[16]('0.07'),
+    value: colorsExperimental.blackAlpha[7],
```

Comment:
```suggestion
    value: colorsExperimental.blackAlpha[6],
```

Looks a smidge dark with `7`